Makefile

```diff
@@ -43,7 +43,23 @@ sle15_activation: check
 	cat package/suse-migration-sle15-activation-spec-template \
 		| sed -e s'@%%VERSION@${version}@' \
 		> dist/suse-migration-sle15-activation.spec
-	tar -czf dist/suse-migration-sle15-activation.tar.gz grub.d
+        # include grub.d dir in MANIFEST
+	sed -ie s'@prune grub.d@graft grub.d@' MANIFEST.in
```

Comment:
There seems to be space vs. tab issues. Please use tabs exclusively in the Makefile. Thanks

Reply:
Left some comments without tabs.Fixed.